src/libraries/Microsoft.PowerFx.Core/Lexer/TexlLexer.cs

```diff
@@ -599,39 +427,39 @@ public string RemoveWhiteSpace(string text)
         }
 
         // Enumerate all supported unary operator keywords.
-        public string[] GetUnaryOperatorKeywords() => _unaryOperatorKeywords;
+        public static string[] GetUnaryOperatorKeywords() => _unaryOperatorKeywords.ToArray();
 
         // Enumerate all supported binary operator keywords.
-        public string[] GetBinaryOperatorKeywords() => _binaryOperatorKeywords;
+        public static string[] GetBinaryOperatorKeywords() => _binaryOperatorKeywords.ToArray();
```

Comment:
>.ToArray() [](http://example.com/codeflow?start=85&length=10)

Does this need to be an array? or can we return the ReadOnlyList?

Else, this is allocating an array on each call.